python/tk_multi_snapshot/snapshot.py

```diff
@@ -430,10 +430,8 @@ def show_snapshot_dlg(self):
             msg = "Unable to snapshot!\n\nPlease save the scene as a valid work file before continuing"
             QtGui.QMessageBox.information(None, "Unable To Snapshot!", msg)
 
-            # try to launch "Shotgun Save As" command if we have it:
-            save_as_cmd = tank.platform.current_engine().commands.get(
-                "Shotgun Save As..."
-            )
+            # try to launch "SG Save As" command if we have it:
+            save_as_cmd = tank.platform.current_engine().commands.get("SG Save As...")
```

Comment:
```suggestion
            save_as_cmd = (
                 tank.platform.current_engine().commands.get("SG Save As...") or
                 tank.platform.current_engine().commands.get("Shotgun Save As...")
            )
```

This way we ensure that someone who hasn't upgraded the workfiles app yet will still be able to work.

Reply:
Note, I haven't tested the line, so you might have to fix it.